### ISSUE TITLE
MSID: 89485 Version: 1 DOI: 2023.06.30.546984

### ISSUE THREAD

**Issue by nlisgo** [elifesciences/enhanced-preprints-import#1055]
[PLACE MANUSCRIPT AND EDITOR DETAILS HERE WHEN AVAILABLE] See step 3

[PLACE PDF URL HERE WHEN AVAILABLE] See step 7

## Step 1. Inform bioRxiv

Who can help: @QueenKraken, @nlisgo, @scottaubrey

- [ ] Manuscript is in data hub index (https://data-hub-api.elifesciences.org/enhanced-preprints/docmaps/v1/by-publisher/elife/get-by-manuscript-id?manuscript_id=89485)

or (only one should be ticked. remove other from description.)

- [ ] Confirmed with Ted at bioRxiv that MECA archive is available

Send the following email to Ted and wait for his reply.

```
Hi Ted,

Please can you prepare the preprint MECA for 10.1101/2023.06.30.546984

Thanks
```

## Step 2. Create preview of manuscript

Who can help: @fred-atherden, @nlisgo, @scottaubrey

- [ ] Manuscript is available for preview (https://prod--epp.elifesciences.org/preview/89485/v1)
    - [ ] Trigger import for staging: https://prod--epp.elifesciences.org/import
    - [ ] Sync data folder with s3://prod-elife-epp-data/data
    - [ ] Trigger import for staging: https://staging--epp.elifesciences.org/import
    - [ ] Sync data folder with s3://staging-elife-epp-data/data
    - [ ] Pull request created to [enhanced-preprint-data](https://github.com/elifesciences/enhanced-preprints-data/pulls)
    - [ ] Post link to pull request below

Pull request: [PLACE LINK TO PULL REQUEST HERE]

Detailed instructions: https://github.com/elifesciences/enhanced-preprints-data#add-a-manuscript

## Step 3: Awaiting public reviews and QC

Who can help: Production team

- [ ] QC passed or QC not needed

Who can help: Editorial team

- [ ] Public reviews available on sciety (https://sciety.org/articles/activity/10.1101/2023.06.30.546984)
- [ ] Manuscript and editor details at the top of this issue (Supplied by Editorial team)
- [ ] Reviews returned by EPP - https://prod--epp.elifesciences.org/api/reviewed-preprints/89485/v1/reviews

<details>
<summary>Example</summary>

```
"msas": "Genetics and Genomics", "Neuroscience"
"msid": "84628"
"version": "1"
"preprintDoi": "10.1101/2022.10.28.514241"
"articleType": "Reviewed Preprint"
"status": "Published from the original preprint after peer review and assessment by eLife."

"Reviewed Preprint posted": "2023-01-02"
"Sent for peer review": "2022-10-28"
"Posted to bioRxiv": "2022-11-21" (link: "Go to bioRxiv": "https://www.biorxiv.org/content/10.1101/2022.10.28.514241v1")

Editors:
Reviewing Editor
Michael B Eisen
University of California, Berkeley, United States
Senior Editor
Michael B Eisen
University of California, Berkeley, United States
```

</details>

## Step 4: Modify manuscripts.json (no PDF)

- [ ] Reviewed preprint is published to EPP (https://elifesciences.org/reviewed-preprints/89485)
    - [ ] Pull request created to [enhanced-preprint-client](https://github.com/elifesciences/enhanced-preprints-client/pulls)
    - [ ] Pull request switched to "Ready for review"
    - [ ] Post link to pull request below
- [ ] Request that a doi be registered
    - [ ] Post link to [#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77) comment below

Pull request: [PLACE LINK TO PULL REQUEST HERE]
[#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77) comment thread: [PLACE LINK TO COMMENT HERE]

<details>
<summary>Instructions to modify manuscripts.json</summary>

- Visit: https://github.com/elifesciences/enhanced-preprints-client/actions/workflows/publish-manuscript.yaml
- Click: Run workflow
- Complete the form and click "Run workflow"
- A successful run should result in a new pull request at https://github.com/elifesciences/enhanced-preprints-client/pulls
- Open the pull request and click the "Ready for review" button to trigger tests
- Once the tests pass and you are happy with the changes the PR can be merged

Example pull request: https://github.com/elifesciences/enhanced-preprints-client/pull/334/files

Once the pull request is merged in it should be available a few minutes later.

</details>

### Request that a doi

Post the following in [#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77):

```
@Fred can you register a doi for https://elifesciences.org/reviewed-preprints/89485
```

## Step 5: Awaiting search reindex

- [ ] Reviewed preprint is avaliable on journal homepage (https://elifesciences.org)

The search reindex is triggered once an hour. We need the reviewed preprint to be indexed as the search application serves the journal homepage.

<details>
<summary>Additional info</summary>

If needed, the jenkins pipeline to reindex search can be triggered sooner.

https://alfred.elifesciences.org/job/process/job/process-reindex-reviewed-preprints/

</details>

## Step 6: Published! Request PDF generation

- [ ] Post the link to the PDF url at the top of the issue
    - [ ] Post link to [#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77) comment below

[#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77) comment thread: [PLACE LINK TO COMMENT HERE]

Post the following to the [#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77) on slack:

```
@Ryan Dix-Peek please can you generate a PDF for https://elifesciences.org/reviewed-preprints/89485
```

## Step 7: Introduce PDF to data folder and git repo

- [ ] PDF is avaliable at https://github.com/elifesciences/enhanced-preprints-data/raw/master/data/89485/v1/89485-v1.pdf

Detailed instructions: https://github.com/elifesciences/enhanced-preprints-data#add-a-pdf

## Step 8: Add PDF url to manuscripts.json

- [ ] Reviewed preprint PDF is available for download (https://prod--epp.elifesciences.org/reviewed-preprints/89485)
    - [ ] Pull request created to [enhanced-preprint-client](https://github.com/elifesciences/enhanced-preprints-client/pulls)
    - [ ] Pull request switched to "Ready for review"
    - [ ] Post link to pull request below

[PLACE LINK TO PULL REQUEST HERE]

<details>
<summary>Instructions to add PDF url to manuscripts.json</summary>

- Visit: https://github.com/elifesciences/enhanced-preprints-client/actions/workflows/add-pdf-url-to-manuscript.yaml
- Click: Run workflow
- Complete the form and click "Run workflow"
- A successful run should result in a new pull request at https://github.com/elifesciences/enhanced-preprints-client/pulls
- Open the pull request and click the "Ready for review" button to trigger tests
- Once the tests pass and you are happy with the changes the PR can be merged

Example pull request: https://github.com/elifesciences/enhanced-preprints-client/pull/397/files

Once the pull request is merged in it should be available a few minutes later.

</details>

## Step 9: Done!

- [ ] Kettle is on!

**Comment by JGilbert-eLife** [comment]
Reviews are not available yet.

Not quite sure what's up with the corresponding authors here. Both first and last authors are corresponding, but it looks like on the preprint, two emails were provided for the last author and this info is applied to both corresponding authors.

![Image](https://github.com/elifesciences/enhanced-preprints-import/assets/1785305/608062df-6041-4e61-9c9b-7d7a740ac0a1)

EPP result:

![Image](https://github.com/elifesciences/enhanced-preprints-import/assets/1785305/3e7c003d-eb46-4001-88be-4ee384c6a34f)

Also, eLocations missing from reference list.

**Comment by fred-atherden** [comment]
Good spot - thanks James. It also looks like `Kevin A. Rucci` did the submitting and is marked as corresponding author in eJP, which conflicts with the PDF. Please could you check with the authors, and ask them to update the preprint?

**Comment by JGilbert-eLife** [comment]
I have reached out to the authors.

**Comment by JGilbert-eLife** [comment]
The authors have updated their preprint, so that the PDF now shows both corresponding authors' email addresses, but the online view on bioRxiv now only shows the last author as corresponding.

https://www.biorxiv.org/content/10.1101/2023.06.30.546984v2

Can we work with this? We'd have all the information we'd need to correct this in our version . . .

**Comment by fred-atherden** [comment]
Nice - thanks James.

We'll need to wait until the MECA package is available to confirm. Hopefully bioRxiv's typesetters capture the info as per the PDF

**Comment by fred-atherden** [comment]
(MECA package/full text still not yet available)

**Comment by fred-atherden** [comment]
MECA package now available at `s3://transfers-elife/biorxiv_Current_Content/September_2023/11_Sep_23_Batch_1731/0b769684-6ed0-1014-8474-d012260c521e.meca`. 

I'll load tomorrow

**Comment by fred-atherden** [comment]
@JGilbert-eLife, I've loaded the latest version for this one - could you take a look when you next have a chance? Thanks

**Comment by JGilbert-eLife** [comment]
Thanks! Looks good - do we need to update the ticket to indicate the bioRxiv v2? (https://www.biorxiv.org/content/10.1101/2023.06.30.546984v2)?

**Comment by fred-atherden** [comment]
Also do the reviews need moving over to the v2 @acollings?

**Comment by acollings** [comment]
No idea, @fred-atherden, but I've moved them across just in case :-)

**Comment by fred-atherden** [comment]
@HazalCiplak - do you know why the reviews aren't in the DocMap for this one as well?

**Comment by acollings** [comment]
(Due to being deleted/reposted, will they appear tomorrow?)

**Comment by fred-atherden** [comment]
Ah sorry, yeah of course

**Comment by HazalCiplak** [comment]
I have rerun the pipelines to got DocMaps available later today.

**Comment by fred-atherden** [comment]
Thanks! I think we're good to go with this one.
